api/openapi.generated.yml

```diff
@@ -43,14 +43,14 @@ paths:
                     description: The HTTP status code
                   pagination_info:
                     description: The pagination information for paginated endpoints
-                    type:
+                    type: &id001
```

Comment:
Is this and the other changes due to the `APIFlask` upgrade?

Reply:
I'm guessing it is something in the underlying dependencies. APIFlask uses apispec to generate this, and it seems PyYAML is what it uses. That said, don't see anything in those underlying libraries recently that would cause a change, so really not sure what is up, but this does appear to be valid yaml